client/components/BookshelfActions.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Book } from 'types/book';
+import { useMutation } from 'react-query';
+import fetcher from 'network/fetcher';
+import Button from '@mui/material/Button';
+import ButtonGroup from '@mui/material/ButtonGroup';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import BookmarkBorderIcon from '@mui/icons-material/BookmarkBorder';
+import BookmarkIcon from '@mui/icons-material/Bookmark';
+
+export interface Props {
+  bookData: Book;
```

Comment:
```suggestion
  book: Book;
```

bookData is a verbose and redundant prop name